src/components/IntervalSlider.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Slider from "@material-ui/core/Slider";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: 450,
+  },
+  margin: {
+    height: theme.spacing(3),
+  },
+}));
+
+const marks = [
```

Comment:
seems a bit repetitive.
a programatic way to do this could be: 

    [1,2,3,4,5,6,7,8,9,10,11,12].map(x=>x*5).map(x=>{return {'value':x, 'label':`{${x}m`}})

although I think your's is more readable :)